towhee/__init__.py

```diff
@@ -107,6 +109,7 @@ def pipeline(task: str, fmc: FileManagerConfig = FileManagerConfig(), branch: st
         yaml_path = fm.get_pipeline(task, branch, force_download)
 
     engine = Engine()
+    engine.framework = framework
```

Comment:
`framework` is a pipeline attribute, if we put it into `engine`, it will be wrong if I run two or more pipelines with different frameworks.

Reply:
I add a step in `towhee/__init__.py`,  every time we start a new pipeline, we will set the `framework` for `engine`.

Reply:
Can we modify the contents of the `GraphRepr` passed to the `Engine` instead of modifying the `Engine` itself?

Reply:
`GraphRepr` only contains the information of the YAML, if we want to specify the framework in `pipeline()`,  we cannot avoid modifying `Engine`.

Reply:
> I add a step in `towhee/__init__.py`, every time we start a new pipeline, we will set the `framework` for `engine`.

What if we start new pipelines in different threads at the same time?

Reply:
Shall we introduce a lock for such behavior?

Reply:
Can we put it into `Pipeline`? Different pipelines can have their own framework info.

Reply:
Oops, that's a bad idea, or shall we change the `_pipeline` list in `Engine` to a dict to store the framework for each pipeline.

Reply:
Putting it into `Pipeline` cannot pass the framework arg to op_loader